src/views/Workflow.vue

```diff
@@ -54,14 +66,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <script>
 import { mixin } from '@/mixins'
 import { datatree } from '@/mixins/treeview'
+import { datatable } from '@/mixins/tableview'
 import { mapState } from 'vuex'
 import Lumino from '@/components/cylc/workflow/Lumino'
-import { WORKFLOW_TREE_DELTAS_SUBSCRIPTION } from '@/graphql/queries'
+import { WORKFLOW_TREE_DELTAS_SUBSCRIPTION, WORKFLOW_TABLE_DELTAS_SUBSCRIPTION } from '@/graphql/queries'
```

Comment:
Both subscriptions are identical. We should be able to use just the tree one, without having to add a new subscription.

![image](https://user-images.githubusercontent.com/304786/118886978-ceea1600-b94d-11eb-8595-e4fb7b5c6aca.png)

The only reason why that's not possible, at the moment, seems to be due to #661 

You don't have to worry about this one @giulianoserrao